### ISSUE TITLE
Require TIME_REMAINING field to be filled in before starting to run a submission

### ISSUE THREAD

**Issue by brucehoff** [Sage-Bionetworks/DigitalMammographyChallenge#142]
in the open phase we allow null TIME_REMAINING but in the competitive phase we should require a value to be present.  This prevents us from starting a submission for a team which has exceeded its allotted time.

**Comment by tschaffter** [comment]
As mentioned in another issue, we should allow submission with no time quota left to start if there is a job slot free. The submission would however be stopped immediately if a a job from a different team who has time quota left is submitted.
